src/components/App.js

```diff
@@ -1,28 +1,60 @@
-import React, { useState, useEffect } from 'react';
+import React, { useState, useEffect } from "react";
+
+import { 
+    getProducts 
+} from "../api";
+
+import { 
+    Product, 
+    SingleProduct, 
+    Cart,
+    NavBar
+} from "./";
+
+import {
+  BrowserRouter as Router,
+  Route,
+  Switch,
+  Redirect,
+} from "react-router-dom";
 
 import {
-  getSomething
-} from '../api';
+    getLocalToken
+} from "../util";
 
 const App = () => {
-  const [message, setMessage] = useState('');
+    const [products, setProducts] = useState([]);
+    const [token, setToken] = useState('');
+    const [user, setUser] = useState({});
 
   useEffect(() => {
-    getSomething()
-      .then(response => {
-        setMessage(response.message);
-      })
-      .catch(error => {
-        setMessage(error.message);
-      });
-  });
+    getProducts().then(setProducts);
```

Comment:
Create another async function that fetches the products and sets the products state.  Then it can just be called in the useEffect or elsewhere.  LIke `useEffect(fetchProducts, []);`